core/domain/stats_services_test.py

```diff
@@ -264,6 +264,11 @@ def test_record_answer_without_retrieving_it_first(self):
             'params': {}
         }])
 
+        sample_answers = stats_services.get_sample_answers(
```

Comment:
Can we have tests written that are dedicated to ``get_sample_answers`` rather than piggybacking on existing tests? Since it's a new function, it probably should have its own set of tests.